packages/mds-policy-service/service/helpers.ts

```diff
@@ -22,11 +26,14 @@ export function translateIntentToPolicy<I extends INTENT_TYPE>(draft: IntentDraf
     draft.policy_fields.start_date = now() + TWENTY_MINUTES + POLICY_START_DATE_FUDGE_FACTOR
   }
 
-  const rule_constants = INTENT_RULE_CONSTANTS[draft.intent_type]
+  const rules = Array.isArray(draft.rule_fields)
```

Comment:
If I’m taking this correctly, I think you could simplify this with `asArray` from mds-utils…
```ts
const rules = asArray(draft.rule_fields).map(…)
```